history/cultures/twilights_hammer.txt

```diff
@@ -23,6 +23,7 @@
 	discover_innovation = innovation_ledger 
 	discover_innovation = innovation_crop_rotation 
 	#
+	discover_innovation = innovation_war_camels
```

Comment:
They have it before getting to Silithus.
Though there are no camels in Silithus.